database/postgres/getIds.js

```diff
@@ -0,0 +1,20 @@
+const md5 = require('md5');
+
+const keys = ['description', 'info', 'similarItems', 'configuration'];
+let [,,input] = process.argv;
```

Comment:
what is going on here? you are assigning "process.argv" to a variable name called "[,,input]"? what is "[,,input]"?